spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe ApplicationHelper do
+
+  describe "#sortable_heading_for" do
+    subject { helper.sortable_heading_for(attribute) }
+    let(:attribute) { "dongle" }
+
+    before do
+      stub_request = Object.new
+      allow(stub_request).to receive(:path).and_return("x")
+      allow(stub_request).to receive(:query_parameters).and_return({sort_direction: "asc"})
+      allow(helper).to receive(:request).and_return(stub_request)
```

Comment:
I think we should enable verification of partial doubles.

RSpec.configure do |config|
  config.mock_with :rspec do |mocks|
    mocks.verify_partial_doubles = true
  end
end